### PR TITLE
jonesdao - update for jglp and jusdc vaults

### PR DESCRIPTION
**Pull request by schroyar**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. The protocol is usually listed within 24 hours of merging the PR
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/jones-dao exports TVL: 
        
 
 
    arbitrum-pool2            6.53 M
    arbitrum-staking          7.74 M
    arbitrum                  8.07 M
    pool2                     6.53 M
    staking                   7.74 M
    
    total                    8.07 M

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "ethers": "^5.6.5",
     "graphql-request": "^4.0.0",
     "hi-base32": "^0.5.1",
+    "js-sha3": "^0.8.0",
```
Comment (g1nt0ki) [inline package.json:34]: I am not really sure why this is needed

**Comment by llamatester** [comment]
The adapter at projects/jones-dao exports TVL: 
        
 
 
    arbitrum-staking          9.04 M
    arbitrum-pool2            6.62 M
    arbitrum                  15.47 M
    staking                   9.04 M
    pool2                     6.62 M
    
    total                    15.47 M

**Comment by llamatester** [comment]
The adapter at projects/jones-dao exports TVL: 
        
 
 
    arbitrum-staking          9.07 M
    arbitrum-pool2            6.65 M
    arbitrum                  15.50 M
    staking                   9.07 M
    pool2                     6.65 M
    
    total                    15.50 M

**Comment by g1nt0ki** [comment]
@schroyar sorry, I missed your comment, already refactored the code here: https://github.com/DefiLlama/DefiLlama-Adapters/commit/3aa4e92f2564e05007ffaac98f2dc797c84d5d01

**Comment by g1nt0ki** [comment]
*it includes your glp changes

**Pull request closed without merging**